.changeset/new-ducks-prove.md

```diff
@@ -0,0 +1,77 @@
+---
+'@razorpay/blade': minor
+---
+
+feat(blade): added bottomsheet component
+
+
+### Migration guide from prerelease version
+
+Update the imports:
```

Comment:
Probably put this migration guide for pre-release under collapsible

Reply:
Also add a note to refer to the installation guide for native consumers because of new native deps

Reply:
Sure